bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -336,6 +335,8 @@ public void dispose () {
 					trackingLock = null;
 				}
 			}
+		} finally {
```

Comment:
Is this change related to the removal of the synchronization or just better style?

Reply:
While it probably not any usual usecase to dispose() Device i wanted to make sure that disposed flag is updated as the very last statement during dispose(). One could also change it to be the very first statement- in any case if isDisposed() is checked in any other thread it has almost no meaning as it could be disposed in the next blink of an eye. So it does not really matter.